### PR TITLE
add: avatar component added

### PR DESCRIPTION
**Pull request by saifudinbadani**

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    z-index: 1;
+    position: absolute;
+    background-color: var(--black-500);
+    border: 2px solid white;
+    height: 1.5rem;
+    width: 1.5rem;
+    border-radius: 50%;
+    bottom: 1.5rem;
+    left: 8rem;
+}
```
Comment (akash-1318) [inline components/avatar/avatar.css:69]: .avatar-online,  .avatar-dnd and .avatar-offline for these classes you are repeating the same code. You only have to change the background color so just add a new class for that.

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
+@import "../../css/common.css";
+
+.avatar{
+
+}
```
Comment (amirnathani9) [inline components/avatar/avatar.css:5]: the class has no value. if it is of no use remove it

File: components/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+.avatar-xs > img{
+    height: 3rem;
+}
+
+/* avatar badges */
+
+.avatar-online{
+    z-index: 1;
+    position: absolute;
+    background-color: var(--green-500);
```
Comment (amirnathani9) [inline components/avatar/avatar.css:38]: dono give the color name to variable instead use primary secondary bg color etc

File: components/avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    border: 2px solid white;
```
Comment (amirnathani9) [inline components/avatar/avatar.css:39]: define the cariable name instead of directly using color

File: components/avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    width: 1.5rem;
+    border-radius: 50%;
+    bottom: 1.5rem;
+    left: 8rem;  
+}
+
+.avatar-dnd{
+    z-index: 1;
+    position: absolute;
+    background-color: var(--red-500);
```
Comment (amirnathani9) [inline components/avatar/avatar.css:50]: donot give the variable name of color

File: components/badge/badge.css [hunk 6]
```diff
@@ -0,0 +1,25 @@
+@import "../../css/common.css";
+
+
+
+.badge-notification{
+    position: absolute;
+    color: white;
+    border-radius: 50%;  
+    background-color: var(--red-500);
```
Comment (amirnathani9) [inline components/badge/badge.css:9]: same mistake here

**Comment by amirnathani9** [comment]
All thing looks good.
-Add discription of Tittle, discription, screenshots and preview link. 
-use the variable for color everywhere and donot give the color name to variable

**Pull request merged**